projects/plugins/jetpack/class.jetpack.php

```diff
@@ -4331,7 +4331,7 @@ function ( $domains ) {
 						exit;
 					}
 
-					if ( static::is_active() && static::connection()->is_user_connected() ) {
+					if ( static::connection()->is_connected() && static::connection()->is_user_connected() ) {
```

Comment:
I was in doubt about this one, but as we talked, the check for `is_user_connected` will require a connection anyway, so it's almost a redundant check. So this looks fine